### PR TITLE
Bring back bundle-size comments and the fox

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by chee** [comment]
that's very nice

**Comment by JakeChampion** [comment]
Not working yet:
```
[11:14:05] Running Karma tests [started]
[11:14:05] Running Karma tests [skipped]
[11:14:05] → No test files found, consider adding some tests to make sure the code is working as expected.
25h25hCommand failed with exit code 2: origami-bundle-size
 ›   Error: Could not get bundle sizes: Cound not get bundle information for 
 ›   o-date at version undefined.
 ›   Build service: Unable to load non-encoded bundle from 
 ›   https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-date%4
 ›   0undefined (status: 404)..
 ›   Repo data: Repo data requires a valid semver version..
o-date bundle size difference from undefined to 3b9d315c632f11960988c4a4122d7988ce1ea58f
```

**Comment by chee** [comment]
what's this: `o-date%4`, o-dateEOT ???

File: .github/workflows/test-origami-component.yml [hunk 1]
```diff
@@ -9,4 +9,10 @@ jobs:
       with:
         node-version: 12.x
     - run: npm install --only=dev
-    - run: npx origami-ci branch
\ No newline at end of file
+    - name: Repository name
+      id: repo_name
+      run: echo ::set-output name=REPO_NAME::$(echo $GITHUB_REPOSITORY | cut -d / -f 2)
```
Comment (chee) [inline .github/workflows/test-origami-component.yml:14]: what shell are we in here? if it's bash we can do `name=REPO_NAME::${GITHUB_REPOSITORY/*\//}`

**Comment by JakeChampion** [comment]
o-date bundle size difference from 4.0.3 to 07237b7490e36c7b577f9a2bf065bdd171b11559
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.3 to ab0877b1ba6965574ab358a30d572ca1ff929935
No significant bundle size differences found.

**Comment by JakeChampion** [comment]
Thank you foxy

File: .github/workflows/test-origami-component.yml (hunk 1 above)
Comment (JakeChampion) [reply .github/workflows/test-origami-component.yml:16]: Let's try it
```suggestion
      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
```

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.3 to 2a1fa4e5b241a4502ac85ecc833d5073617a9339
No significant bundle size differences found.

**Comment by JakeChampion** [comment]
Still getting this...
```
Running Karma tests [skipped]
→ No test files found, consider adding some tests to make sure the code is working as expected.
```

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.3 to fc98560bbd5d7e185e6fc68af6ea003d4a45e01f
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-date bundle size difference from 4.0.3 to 6ff3ad961ec44899800da23863e0909a913da76d
No significant bundle size differences found.

**Pull request merged**